### DISCUSSION TITLE
How can I ignore ssl certificate issues on deno delpoy?

### DISCUSSION THREAD

**Discussion by mwstr** [denoland/deno#13208] [Q&A] [answered]
`--unsafely-ignore-certificate-errors` is unusable in this situation...

**Comment by lucacasonato** [comment]
Why do you want to do this?

**Reply by mwstr** [reply]
Because of `error trying to connect: invalid certificate: UnknownIssuer` on fetch

**Reply by ivanpajon** [reply]
I'm facing the same issue

**Comment by kitsonk** [comment, marked as answer]
Deploy issues and feedback should occur on `deploy_feedback`.  I have created https://github.com/denoland/deploy_feedback/issues/175 there.
